python/repositories.bzl

```
@@ -142,6 +142,12 @@ filegroup(
         ],
         exclude = [
             "**/* *", # Bazel does not support spaces in file names.
+            # static libraries
+            "lib/*.a",
+            "lib/**/*.a",
+            # tests for the standard libraries.
+            "lib/**/test/**",
+            "lib/**/tests/**",
```

```suggestion
            "lib/python{python_version}/**/test/**",
            "lib/python{python_version}/**/tests/**",
```

Let me know if I'm missing something, but I think we should only expect test files under that specific folder.

---

Verified that same tests for standard library were removed. Realized tests for `idlelib` (`lib/python{python_version}/idlelib/idle_test`), but removing the tests can be another PR.